docs/modules/plugin/pages/tips-and-tricks.adoc

```diff
@@ -0,0 +1,83 @@
+= Tips & Tricks
+
+== Fail on missing attributes
+
+Attributes whose value cannot be resolved are ignored by default and not shown as error or warning.
+If you need to, this behaviour can be modified using the https://asciidoctor.org/docs/user-manual/#missing-attribute[missing-attribute] attribute.
+
+Combining it with logHandler option it is possible to report an error and abort a build in case of missing attributes.
+
+[source, xml]
```

Comment:
I recommend being consistent about using a space after the comma. I prefer it without a space, but as long as it's consistent, that's what really matters.

Reply:
100% agree, thanks for pointing it